### ISSUE TITLE
NMM: v65.2 I cannot connect to verify version or donwload via manager

### ISSUE THREAD

**Issue by XeroStaticFlux** [Nexus-Mods/Nexus-Mod-Manager#640]
I updated the application to the current available (65.2) however it still gives me the following error:
https://ibb.co/khN3eU

Can someone please help me figure out my problem with this? 
![wtff](https://user-images.githubusercontent.com/41937292/46578521-8960c080-c9d0-11e8-8d89-3ae772bf746d.jpg)

**Comment by squid-box** [comment]
Grab the [actual latest version](https://github.com/Nexus-Mods/Nexus-Mod-Manager/releases) and this will be fixed.

**Comment by XeroStaticFlux** [comment]
I had downloaded from the official link off the nexus website, and unless another was available somewhere else it wasnt listed or mentioned at the time.  Regardless i fixed it myself sorta (with this version) after farting around with some settings.   I forget which i changed but clicked a button in the settings at some point and it all the sudden auto connected.  It resets each time I load it but works I got what I needed installed for now.  Next time i need to add a new mod ill update, yet again.
